src/serverConfig.ts

```diff
@@ -25,6 +27,10 @@ export const initServer = (): Express => {
         return userController.signIn(req, res);
     });
 
+    app.post('/api/auth/', (req: Request, res: Response) => {
```

Comment:
it doesn't make sense to have. we're going to use validate to create private endpoints.